Data/Sys/GameSettings/GKM.ini

```diff
@@ -15,3 +15,6 @@
 [Video_Settings]
 SafeTextureCacheColorSamples = 512
 
+[Video_Hacks]
```

Comment:
```suggestion
[Video_Settings]
```

Reply:
'Video settings' is already written 3 lines above, so it would be repeated.